From 122457e36334f6555a53897ab5bd482f35a69717 Mon Sep 17 00:00:00 2001
From: Denis Semakin <d.semakin@rt-solar.ru>
Date: Tue, 9 Jun 2020 14:00:53 +0300
Subject: [PATCH] [u-boot-solar] Add FIT signature feature.

Signed-off-by: Denis Semakin <d.semakin@rt-solar.ru>
---
 configs/orangepi_pc_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/orangepi_pc_defconfig b/configs/orangepi_pc_defconfig
index a676294a8b..766164c493 100644
--- a/configs/orangepi_pc_defconfig
+++ b/configs/orangepi_pc_defconfig
@@ -10,3 +10,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_SY8106A_POWER=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_FIT_SIGNATURE=y
